scalableminds/webknossos
# Use less ram when initializing data textures

See code comments.

### URL of deployed dev instance (used for testing):
- https://lessram.webknossos.xyz

### Steps to test:
- open a dataset with a seg layer
- check that console does not show webgl errors
- change the color of a segment
- run `window.testContextLoss()` in the console and check that wk is usable again after a few seconds of flickering

### Issues:
- perf perf perf

------
(Please delete unneeded items, merge only when none are left open)
